src/Catty/ViewController/Continue&New/MaintainObject/MaintainLooks/LooksTableViewController.m

```diff
@@ -644,53 +644,54 @@ - (void)showAddLookActionSheet
-     addDefaultActionWithTitle:kLocalizedDrawNewImage handler:^{
-         dispatch_async(dispatch_get_main_queue(), ^{
-             PaintViewController *vc = [self.storyboard instantiateViewControllerWithIdentifier:kPaintViewControllerIdentifier];
-             vc.delegate = self;
-             vc.editingPath = nil;
-             vc.projectHeight = self.object.scene.project.header.screenHeight.floatValue;
-             vc.projectWidth = self.object.scene.project.header.screenWidth.floatValue;
-             NSNumber *value = [NSNumber numberWithInt:UIInterfaceOrientationPortrait];
-             [[UIDevice currentDevice] setValue:value forKey:@"orientation"];
-             [self.navigationController pushViewController:vc animated:YES];
-         });
-     }]
-     addDefaultActionWithTitle:kLocalizedMediaLibrary handler:^{
-         dispatch_async(dispatch_get_main_queue(), ^{
-             if (self.object.isBackground) {
-                 [self showBackgroundsMediaLibrary];
-             } else {
-                 [self showLooksMediaLibrary];
-             }
-         });
-     }] build]
+        addDefaultActionWithTitle:kLocalizedDrawNewImage handler:^{
+        dispatch_async(dispatch_get_main_queue(), ^{
+            PaintViewController *vc = [self.storyboard instantiateViewControllerWithIdentifier:kPaintViewControllerIdentifier];
+            vc.delegate = self;
+            vc.editingPath = nil;
+            vc.projectHeight = self.object.scene.project.header.screenHeight.floatValue;
+            vc.projectWidth = self.object.scene.project.header.screenWidth.floatValue;
+            NSNumber *value = [NSNumber numberWithInt:UIInterfaceOrientationPortrait];
+            [[UIDevice currentDevice] setValue:value forKey:@"orientation"];
+            [self.navigationController pushViewController:vc animated:YES];
+        });
+    }]
+       addDefaultActionWithTitle:kLocalizedMediaLibrary handler:^{
+        dispatch_async(dispatch_get_main_queue(), ^{
+            if (self.object.isBackground) {
+                [self showBackgroundsMediaLibrary];
+            } else {
+                [self showLooksMediaLibrary];
+            }
+        });
+    }] build]
      showWithController:self];
 }
 
 - (void)suggestToOpenSettingsAppWithMessage:(NSString *)message {
     [[[[[AlertControllerBuilder alertWithTitle:nil message:message]
-     addCancelActionWithTitle:kLocalizedCancel handler:nil]
-     addDefaultActionWithTitle:kLocalizedSettings handler:^{
-           NSDebug(@"Settings Action");
-           [[UIApplication sharedApplication] openURL:[NSURL URLWithString:UIApplicationOpenSettingsURLString] options:@{} completionHandler:nil];
-     }] build]
+        
+        addCancelActionWithTitle:kLocalizedCancel handler:nil]
+       addDefaultActionWithTitle:kLocalizedSettings handler:^{
+        NSDebug(@"Settings Action");
+        [[UIApplication sharedApplication] openURL:[NSURL URLWithString:UIApplicationOpenSettingsURLString]];
```

Comment:
This line gives a compiler warning:
`'openURL:' is deprecated: first deprecated in iOS 10.0`

Please bring back the initial `openURL` call (with `options` and `completionHandler` paramaters).